src/main/java/jp/co/future/uroborosql/UroboroSQL.java

```diff
@@ -102,6 +103,7 @@ public static UroboroSQLBuilder builder(final DataSource dataSource) {
 		private SqlAgentFactory sqlAgentFactory;
 		private EntityHandler<?> entityHandler;
 		private Dialect dialect;
+		private InsertsType defaultInsertsType;
```

Comment:
Should defaultInsertsType be kept in SqlAgentFactory?

Reply:
I moved `defaultInsertsType` to `SqlAgentFactory`.

Reply:
OK
I think it's good